### PR TITLE
Distribution improvements

### PR DESCRIPTION
**Pull request by frant-hartm**
*  Set the hazelcastDownloadId to `distribution` for tar.gz distribution
5ac586b 

* Add JET_MODULES variable to add jars from opt folder to classpath

### CODE REVIEW

**Comment by hz-highfive** [comment]
The job `jet-oss-pr-builder-docs-no-build` of your PR failed ([log](
https://j-artifacts.s3.amazonaws.com/jet-oss-pr-builder-docs-no-build/168/build_log.txt), [artifacts](https://j-artifacts.s3.amazonaws.com/jet-oss-pr-builder-docs-no-build/168/jet-oss-pr-builder-docs-no-build-168.zip)). 
Through arcane magic we have determined that the following fragments from the build log may contain information about the problem.
<details>
<summary>Click to expand the log file</summary>
<pre>
--------------------------
---------SUMMARY----------
--------------------------
There are unexpected files.
Unexpected file: hazelcast-jet-distribution/src/bin-regular/common.sh
</details>
</pre>

**Comment by cangencer** [comment]
verify

File: site/docs/operations/docker.md [hunk 1]
```diff
@@ -278,6 +278,20 @@ To see the default content of the files run the following command:
 docker run hazelcast/hazelcast-jet cat /opt/hazelcast-jet/config/hazelcast-jet.yaml
 ```
 
+## Use Built-in Modules
```
Comment (cangencer) [inline site/docs/operations/docker.md:281]: https://jet-start.sh/docs/operations/installation is it worth expanding this page as well? right now it doesn't talk so much about how to use built in modules.

**Comment by cangencer** [comment]
I tested and was not able to see Hazelcast/Jet metrics on the prometheus port, even though I can see them with `jconsole`.

**Pull request merged**